### PR TITLE
Turns README link to a Markdown link

### PR DESCRIPTION
**Pull request by marcus-crane**
This is a test change to show how PRs work.

You can comment, review and all sorts of other neat things 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ninjaStar
 The ninja star activity used for team building exercises. Origami Ninja Stars not included.
 
-http://cloong.ceredin.com/ninjastar.html
+You can view it [http://cloong.ceredin.com/ninjastar.html](here).
```
Comment (marcus-crane) [inline README.md:4]: Maybe this line could do with an exclamation mark or maybe that would be too much

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 I found a creative way of doing a team building exercise. I folded several origami paper ninja stars, and made this webpage which randomises different "getting to know you" questions.
```
Comment (marcus-crane) [inline README.md:6]: Looks good to me!

**Comment by marcus-crane** [comment]
Normally self-reviewing is silly of course. It should be another team member, or the repo owner, who does that.

**Comment by marcus-crane** [comment]
Need to get a review from @Ceredin

File: README.md (hunk 1 above)
Comment (Ceredin) [reply README.md:4]: Haha I think you might be overthinking the punctuation. :) I'm assuming the brackets around the URL will make it into a link?

**Comment by marcus-crane** [comment]
Ah yes, well, you can see why a review is needed because sometimes people do markdown links wrong ;)

**Pull request merged**